OpenRA.Game/Map/MapGrid.cs

```diff
@@ -107,6 +107,7 @@ public class MapGrid : IGlobalModData
 	{
 		public readonly MapGridType Type = MapGridType.Rectangular;
 		public readonly Size TileSize = new(24, 24);
+		public int TileScale { get; }
```

Comment:
Style issues - a property inbetween fields. A not-readonly thing between readonly fields.
This doesn't need to be a property though, it can just be a readonly field, right?

P.S.: Bonus points if you unhardcode the value(s).

Reply:
Made it readonly. It must have been a remanent of a larger scope refactor I pulled it from.

> P.S.: Bonus points if you unhardcode the value(s).

These values are super hardcoded everywhere in the engine. It's why the modular map format refactor is such a difficult task. PR's like this get us one step closer to unhardcoding.